src/app/shared/sync.service.ts

```diff
@@ -64,24 +64,9 @@ export class SyncService {
   private syncParams(opt, credentials, type) {
     const dbSource = opt.dbSource || opt.db;
     const dbTarget = opt.dbTarget || opt.db;
-    const parentDomain = opt.parentDomain || this.userService.getConfig().parentDomain;
-    const code = opt.code || this.userService.getConfig().code;
```

Comment:
we will not have userService when we are doing configuration so we do not have code as well. That's why had kept code on opt.

Reply:
See line 40.  It's included there.

Reply:
sorry @paulbert I'm pointing to code you use on line 95. you need opt for code as well.